nestjs/src/gratitudes/dto/heroes-radar-badge.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Badge } from './badge.dto';
+
+export interface HeroesRadarBadge {
+  badgeId: Badge;
+  badgeCount: number;
+}
+
+export class HeroesRadarBadgeDto {
+  constructor(badge: HeroesRadarBadge) {
+    this.badgeId = badge.badgeId;
+    this.badgeCount = badge.badgeCount;
+  }
+
+  @ApiProperty()
+  public badgeId: Badge;
+
+  @ApiProperty()
+  public badgeCount: number;
```

Comment:
consider omitting `badge` in the names of properties

Reply:
> consider omitting `badge` in the names of properties

Thank you for dispelling my doubts. I was thinking about that too

Reply:
Fixed